lessons/basics/mix.md

```diff
@@ -7,7 +7,7 @@ order: 9
 lang: en
 ---
 
-Before we can dive into the deeper waters of Elixir we first need to learn about mix. If you're familiar with Ruby mix is Bundler, RubyGems, and Rake combined. It's a crucial part of any Elixir project and in this lesson we're going to explore just a few of its great features. To see all that mix has to offer run `mix help`.
+Before we can dive into the deeper waters of Elixir we first need to learn about mix. If you're familiar with Ruby, mix is Bundler, RubyGems, and Rake combined. It's a crucial part of any Elixir project and in this lesson we're going to explore just a few of its great features. To see all that mix has to offer run `mix help`.
```

Comment:
@eksperimental can you compile and run your code without Mix?  As far as I know, you can't have an Elixir project without Mix, thus it's critical to the project.  Please correct me if I'm wrong 😁